common/alert/reports/a11y-1.json

```
@@ -0,0 +1,908 @@
+    },
+    {
+      "@id": "_:sc_1.4.4",
+      "earl:test": "WCAG21:resize-text",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G142",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.5",
+      "earl:test": "WCAG21:image-of-text",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "I have mentioned C22 because it mentioned about using css classes",
+      "earl:result": ["WCAG21Tech:general/C22"],
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:css/C22",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.10",
+      "earl:test": "WCAG21:reflow",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "All other success criterias are checking for 400% zoom level, which is addressed by C32",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:css/C32",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.11",
+      "earl:test": "WCAG21:non-text-contrast",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "G207, is applicable when using div",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G207",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.12",
+      "earl:test": "WCAG21:text-spacing",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "added class='test-textSpacing' to check text is not clipped",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:css/C36",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:css/C35",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:failures/F104",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        },
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
```

To fix the CI errors about JSON validation
```suggestion
        {
            "earl:test": "WCAG2xTech:failures/F104",
            "earl:outcome": "earl:passed",
            "earl:subject": "_:subject"
        }
      ],
      "earl:mode": "earl:manual",
      "@type": ["earl:TestResult", "earl:Assertion"]
```